tests/test_framework.py

```diff
@@ -48,7 +48,8 @@
     'post_call_analysis': 'python tests/pca_test.py',
     'function_calling': 'python tests/fc_test.py',
     'web_crawled_data_retriever': 'python tests/web_crawling_test.py',
-    # 'search_assistant': 'python cli_test.py --query "test query"',
+    'search_assistant': 'python tests/search_assistant_basic_test.py',
+    'search_assistant': 'python tests/search_assistant_rag_test.py',
```

Comment:
@snova-petrojm - Can you find a way to combine the basic_test.py and rag_test.py into one script or command for the framework.py. This is because it expects one command per kit. 

Looking at the above, you could either combine your unit test files, or create a script that runs both. Personally I would advise we combined the rag and basic tests into one unit test file.

Reply:
Merged into one script